notebooks/244-depth-estimation-videpth/244-depth-estimation-videpth.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.            depth_pred = (
</pre><p>do you really need to switch from numpy to pytorch and back for just making resize and squeeze? Is it possible to make openvino prediction fully independent from torch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
`torch.nn.functional.interpolate()` compulsorily takes a tensor as an input and hence the initial conversion. Later, in the tutorial, several masking operations for global scale and shift alignment are carried out which are easily done (and convenient) to carry out on the `np.ndarray` type and hence `depth_pred` involves conversion back and forth from numpy with a `torch.Tensor` as an intermediate.